newIDE/app/src/stories/componentStories/ConfirmDialog.stories.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-anonymous-default-export */
```

Comment:
nice, this is annoyingly impacting all our stories files.
Maybe we could exclude this rule for all the stories folder: https://stackoverflow.com/a/41316801

Reply:
Done, thanks!

Reply:
no, thank you!